pywbem/_subscription_manager.py

```diff
@@ -172,101 +187,141 @@ def __repr__(self):
-        and port components.
-        FUTURE TODO: we could leave off port and use defaults. based on scheme
+        Add the URL of a WBEM listener to  the set of listeners for which
+        indications will be targeted for the defined WBEM Server.
+
+        This function creates a destination instance for each specified listener
+        in listener_urls and registers that instance with the WBEM server
+        defined by server_id.
 
         Parameters:
-          host (:term:`string`):
-            IP address or host name this listener can be reached at.
 
-          port (:term:`string` or :term:`integer`):
-            port the listener can be reached at.
+          server_id (:term:`string`):
+            The server ID for the WBEM server, returned by :meth:`add_server`.
+
+          listener_urls ((list of :term:`string`) or (:term:`string`)):
+            Either a  single listener URL or list of listener
+            urls. This represents the listeners for which indications will
+            be routed and for which destinations instances are registered
+            with the WBEM server.
+
+            Each listener URL is defined in the format:
+
+              ``[{scheme}://]{host}[:{port}]``
+
+            The following URL schemes are supported:
+
+            * ``https``: Causes HTTPS to be used.
+            * ``http``: Causes HTTP to be used. This is the default
+
+            The host can be specified in any of the usual formats:
+
+            * a short or fully qualified DNS hostname
+            * a literal (= dotted) IPv4 address
+            * a literal IPv6 address, formatted as defined in :term:`RFC3986`
+              with the extensions for zone identifiers as defined in
+              :term:`RFC6874`, supporting ``-`` (minus) for the delimiter
+              before the zone ID string, as an additional choice to ``%25``.
+
+            If no port is specified in the URL, the default ports are:
+
+            * If HTTPS is used, port 5989.
+            * If HTTP is used, port 5988.
+
+            See `WBEMConnection` for examples of valid urls.
+
+          owned (:class:`py:bool`)
+            Defines whether a destination whose lifecycle is owned by this
+            server or not is to be created. If True, a filter is
+            created that is defined by the life cycle of this server instance
+            within this subscription manager.  If False a  filter is created
+            that it may outlive the lifecycle of the server instance
+            and must be removed manually by the user (for example using
+            the `remove_filter` function.
+
+        Returns:
+
+            list of :class:`~pywbem.CIMInstanceName`: The indication
+            destination instance paths for the defined listener_urls.
+            `Owned` destinations are recorded in the subscription manager for
+            the target 1server_id` so is not needed by the user. However
```

Comment:
* I don't think we need to mention that it is recorded by server.
* Incorrect markup "1server_id`".
* "so is not needed by the user" is incomplete.